### PR TITLE
scope: Update Diamond Bluff scope a bit

### PR DESCRIPTION
**Pull request by mestery**
Based on some internal discussions in the minimum requirements group,
update the scope to reflect a bit more than simply PCIe cards.

Also, add a note about enabling a rich open source application ecosystem
on top of Diamond Bluff.

Signed-off-by: Kyle Mestery <mestery@mestery.com>

### CODE REVIEW

**Review by krmurph — COMMENTED** [review]
Looks good to me, I added a couple others as reviewers to take a look too

**Comment by mestery** [comment]
Thanks @krmurph, I'll wait to merge it until @timmichels and/or @pdp2shirts review.

File: README.md [hunk 1]
```diff
@@ -89,10 +89,11 @@ The Diamond Bluff project is being created to address these questions and to
 foster the emergence of such an open and creative software eco-system for
 DPU/IPU based cloud infrastructure. The project intends to delineate what a
 DPU/IPU is, to loosely define a framework(s) and architecture for a
-DPU/IPU-based software stack(s) applicable to any vendors hardware solution, to
-integrate with existing open source projects aligned to the same vision such as
-the Linux kernel and[ IPDK.io](https://ipdk.io) and create new APIs for
-interaction with and between the elements of the DPU/IPU ecosystem:
+DPU/IPU-based software stack(s) applicable to any vendors hardware solution,
+allowing the creation of a rich open source application ecosystem, to integrate
```
Comment (timmichels) [inline README.md:93]: , to allow the creation of .....

File: README.md [hunk 2]
```diff
@@ -89,10 +89,11 @@ The Diamond Bluff project is being created to address these questions and to
...
+with existing open source projects aligned to the same vision such as the Linux
+kernel and[ IPDK.io](https://ipdk.io) and create new APIs for interaction with
```
Comment (timmichels) [inline README.md:95]: Another tweak (in a section you didn't change!) - "and to create new APIs ...."

Comment (mestery) [reply README.md:95]: Sure, I'll clean this up as a part of this commit.

**Review by timmichels — CHANGES_REQUESTED** [review]
A couple tweaks for readability

**Review by timmichels — APPROVED** [review]

**Review by pdp2shirts — CHANGES_REQUESTED** [review]
LGTM overall. on line 99 "the host Node" should probably be "the host node"

**Pull request merged**